src/components/MovieDetails/MovieDetails.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import { render, screen } from '@testing-library/react';
+import MovieDetails from './MovieDetails';
+import { MovieDetailsProps } from '../../models';
+
+describe('MovieDetails Component', () => {
+  const defaultProps: MovieDetailsProps = {
+    imageUrl: `./src/assets/Bitmap.png`,
+    title: 'Movie title',
+    rating: 8.8,
+    duration: '2h 34min',
```

Comment:
The value is going to come as a number from BE, so be prepared to convert it to a string by yourself (it is not a problem if you do it later).